src/panels/lovelace/strategies/map/map-view-strategy.ts

```diff
@@ -0,0 +1,58 @@
+import { ReactiveElement } from "lit";
+import { customElement } from "lit/decorators";
+import { computeStateDomain } from "../../../../common/entity/compute_state_domain";
+import { LovelaceViewConfig } from "../../../../data/lovelace/config/view";
+import { HomeAssistant } from "../../../../types";
+import { MapCardConfig } from "../../cards/types";
+
+export type MapViewStrategyConfig = {
+  type: "map";
+};
+
+export const getMapEntities = (hass: HomeAssistant) => {
```

Comment:
Why is this exported?

Reply:
It was used in the editor but it's not used anymore